lib/msal-browser/test/app/PublicClientApplication.spec.ts

```diff
@@ -136,6 +136,75 @@ describe("PublicClientApplication.ts Class Unit Tests", () => {
     });
 
     describe("intialize tests", () => {
+        jest.setTimeout(9999999);
```

Comment:
This is a pretty high timeout, anything smaller we can use?

Reply:
It's just a one-time test to verify that `app.init` calls are serialized by the event loop. I have no plans to merge it.